alf/trainers/policy_trainer.py

```diff
@@ -241,6 +241,20 @@ def _markdownify(paragraph):
             alf.summary.text('diff', _markdownify(git_utils.get_diff()))
             alf.summary.text('seed', str(self._random_seed))
 
+            if self._config.code_snapshots is not None:
+                for f in self._config.code_snapshots:
+                    path = os.path.join(
+                        os.path.abspath(os.path.dirname(__file__)), "..", f)
+                    if not os.path.isfile(path):
+                        common.warning_once(
+                            "The code file '%s' for summary is invalid" % path)
+                        continue
+                    with open(path, 'r') as fin:
+                        code = fin.read()
+                        # adding "<pre>" will make TB show raw text instead of MD
+                        alf.summary.text('code/%s' % f,
+                                         "<pre>" + code + "</pre>")
```

Comment:
Just for your consideration, this snippet of js code shows python syntax highlighting:
https://stackoverflow.com/questions/62029757/nicely-formatted-python-code-in-html-files

Reply:
> Just for your consideration, this snippet of js code shows python syntax highlighting:
> https://stackoverflow.com/questions/62029757/nicely-formatted-python-code-in-html-files

Not sure if tensorboard allows injecting js in its text tab?

Reply:
Just tried TB always interprets the text as markdown.